extensions/multi-display/README.md

```diff
@@ -0,0 +1,21 @@
+<p align="center">
+  <img src="assets/icon.png" height="128" width="128">
+  <h1 align="center">DouTu Extension for Raycast</h1>
```

Comment:
Think this is copied from your other extension

Reply:
I'm so sorry, it's been fixed.